js/stateTitle.js

```diff
@@ -12,7 +12,7 @@ var stateTitle={
         this.title = game.add.tileSprite(0, game.height-640,game.width, 640, 'title');
         this.title.autoScroll(-100,0);
 
-        this.btnStart=game.add.button(110,400,'button',this.startGame,this,0,1,0);
+        this.btnStart=game.add.button(110,400,'button',this.startGame,this,0,1,1);

```

Comment:
So the parameters of `game.add.button` are `x`, `y`, `key`, `callback`, `callbackContext`, and most importantly, `overFrame`, `outFrame`, `downFrame`, and `upFrame`. In order for the button to respond to events when set up this way, the relevant parameters need to have non-zero values. In this case, your button was not responding to the "down" event because it had a zero value for `downFrame`.

I am not as familiar with `game.add.button`, so I'm not sure exactly why this is the case only on mobile. May be worth asking on #phaserjs.